public/theme/htw.css

```diff
@@ -4,6 +4,9 @@ body {
 a {
   color: var(--main-color);
 }
+a svg {
+  fill: var(--main-color);
```

Comment:
you can use this directly profile.ejs and don't need to change the global htw.css

Reply:
ah, you are preparing to add more icons, I see :) yeah, that's a good point to leave it here